### PR TITLE
Docs for PyMongo integration

### PR DESCRIPTION
**Pull request by Agalin**
* Added guide for PyMongo
* Added wizard for PyMongo

Docs for [getsentry/sentry-python/#1590](https://github.com/getsentry/sentry-python/pull/1590)

TBH I wasn't able to test this change, setting up docs build environment is harder than implementing PyMongo integration in Python SDK. 😓 Python 2 is no longer part of the macOS and I'd rather avoid installing it on my machine while it's required due to ancient `node-gyp` version. Latest node container also fails after installing Python 2.7 on hard to debug C code.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: src/platforms/python/guides/pymongo/index.mdx
Comment (imatwawana) [inline src/platforms/python/guides/pymongo/index.mdx:5]: There was no previously existing PyMongo page that we need to redirect from so this isn't necessary.

```suggestion
```

File: src/platforms/python/guides/pymongo/index.mdx
Comment (imatwawana) [inline src/platforms/python/guides/pymongo/index.mdx:9]:
```suggestion
The Pymongo integration adds support for [Pymongo](https://www.mongodb.com/docs/drivers/pymongo/), the official MongoDB
```

File: src/platforms/python/guides/pymongo/index.mdx
Comment (imatwawana) [inline src/platforms/python/guides/pymongo/index.mdx:22]:
```suggestion
To configure the SDK, initialize it before creating any of PyMongo's MongoClient instances:
```

File: src/platforms/python/guides/pymongo/index.mdx
Comment (imatwawana) [inline src/platforms/python/guides/pymongo/index.mdx:53]:
```suggestion
MongoDB use it under the hood, like, for example `mongoengine` (always) or `umongo` (if selected from multiple available
```

File: src/platforms/python/guides/pymongo/index.mdx
Comment (imatwawana) [inline src/platforms/python/guides/pymongo/index.mdx:56]:
```suggestion
If you use a different driver (for example, `motor`, `TxMongo`), this integration won't work.
```

File: src/platforms/python/guides/pymongo/index.mdx
Comment (imatwawana) [inline src/platforms/python/guides/pymongo/index.mdx:60]:
```suggestion
While `mongomock` can be used to replace `PyMongo` in tests, it doesn't implement all features available in the official
```

**Review by imatwawana — COMMENTED** [review]
I've made some edits to more closely align with our style guidelines and fixed a couple typos.

File: src/platforms/python/guides/pymongo/index.mdx
Comment (Agalin) [reply src/platforms/python/guides/pymongo/index.mdx:5]: Took it from FastAPI (as the most recently added integration), expected to be old addressing still available somewhere.

**Comment by Agalin** [comment]
Thanks. I've also noticed a few inconsistencies in `PyMongo` capitalization, changed all to the one used by its docs (uppercase `P` and `M`).

**Comment by Agalin** [comment]
One more thing. Just realised that list of integrations on the right is provided for "frameworks" according to the main python page. Should PyMongo docs instead be placed to Configuration -> Integrations section? Redis and SQLAlchemy are there - but so is Flask which should rather be considered a framework.

**Comment by imatwawana** [comment]
> One more thing. Just realised that list of integrations on the right is provided for "frameworks" according to the main python page. Should PyMongo docs instead be placed to Configuration -> Integrations section? Redis and SQLAlchemy are there - but so is Flask which should rather be considered a framework.

@getsentry/team-web-sdk-backend - You'll need to answer this.

File: src/platforms/python/guides/pymongo/index.mdx
Comment (polina-popova) [inline src/platforms/python/guides/pymongo/index.mdx:53]: Hi,
we use `motor` in our project, and it also [uses](https://motor.readthedocs.io/en/stable/requirements.html) `pymongo` under the hood, so this integration will work (actually, I checked it by monkey patching the same integration some time ago, so I am quite sure in this)).

Comment (Agalin) [reply src/platforms/python/guides/pymongo/index.mdx:53]: Oh. Interesting. Didn't know it's built on top of pymongo. Expected it to be written from scratch. But indeed it depends on pymongo. Will update this fragment.

**Comment by Agalin** [comment]
Well. Still waiting for decision on docs placement but it doesn't matter as long as integration itself is not reviewed.

**Comment by sl0thentr0py** [comment]
sorry @Agalin, we'll try to fit this in soon, too much going on.

**Comment by antonpirker** [comment]
> One more thing. Just realised that list of integrations on the right is provided for "frameworks" according to the main python page. Should PyMongo docs instead be placed to Configuration -> Integrations section? Redis and SQLAlchemy are there - but so is Flask which should rather be considered a framework.

Imo the guides are a good place. (If you are interested: There is currently a discussion going on about maybe changing the location of some parts of the documentation, but because we have a lot of different languages/platforms that work in different ways this can be a longer endeavor: https://github.com/getsentry/sentry-docs/issues/5669 )

**Comment by antonpirker** [comment]
I have added a screenshot and moved some parts a bit around in your PR, could you merge this in please: https://github.com/operasoftware/sentry-docs/pull/1 @Agalin ?

**Comment by Agalin** [comment]
Added a few minor comments to that one. Keep in mind I still need to update those docs with results of the discussion here (notably about `motor`).

**Comment by antonpirker** [comment]
I have now made the changes @Agalin 
Now my branch is correct with `motor`, right?

**Comment by Agalin** [comment]
You've merged my latest changes so yeah, it's fine. There is one small discussion left in that pull request.

**Comment by antonpirker** [comment]
So now this PR is ready for release. After the Python SDK was released, I will merge this.

**Review by antonpirker — APPROVED** [review]

**Pull request merged**
